src/helpers.rs

```diff
@@ -471,6 +471,15 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
     }
 }
 
+/// Check that the number of args is what we expect.
+pub fn check_args<'a, 'tcx, const N: usize>(args: &'a [OpTy<'tcx, Tag>]) -> InterpResult<'tcx, &'a [OpTy<'tcx, Tag>; N]>
```

Comment:
The function name should reflect that this is just about the argument count... maybe `check_arg_count`?